src/msalexample-ios-macos/Shared/MicrosoftLoginView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+struct MicrosoftLoginView: View {
+    @State private var isAuthenticated = false
+    @State private var graphResult = ""
+
+    var body: some View {
+        VStack {
+            Text("Welcome to the SwiftUI MSAL tutorial for iOS and macOS")
+            Text(graphResult)
+            Button {
+                if (!isAuthenticated) {
+                    MSALAuthentication.signin(completion: { securityToken in
+                        isAuthenticated.toggle()
+
+                        guard let meUrl = URL(string: "https://graph.microsoft.com/v1.0/me") else {
+                            return
+                        }
+
+                        var request = URLRequest(url: meUrl)
+                        request.httpMethod = "GET"
+                        request.addValue("Bearer \(securityToken!)", forHTTPHeaderField: "Authorization")
+
+                        URLSession.shared.dataTask(with: request, completionHandler: { data, response, error in
+                            guard let httpResponse = response as? HTTPURLResponse, httpResponse.statusCode == 200, error == nil else {
+                                print(error!.localizedDescription)
+                                return
+                            }
+
+                            if let json = try? JSONSerialization.jsonObject(with: data!, options: .mutableContainers),
+                               let jsonData = try? JSONSerialization.data(withJSONObject: json, options: .prettyPrinted) {
+                                graphResult = String(decoding: jsonData, as: UTF8.self)
+                            } else {
+                                print("An error has ocurred")
+                            }
+                        }).resume()
+                    })
+                }
+                else {
+                    // IMPORTANT: this require keychain capabilities to be added and signed with a valid development certificate. For more information, please refer
+                    // to https://github.com/AzureAD/microsoft-authentication-library-for-objc/tree/3bc25ad3c38c0f0044e3fc624a841ac4789478c0#macos-only-steps
```

Comment:
This looks important enough to be its own step or at least have a prominent mention in the **Setup** section of the README.

You can link to Apple's how-to and not include the steps themselves, but it should at least be a step of its own. Or, if it's automatic when you open the project in Xcode, note it in the "Open Xcode" step that it is a requirement.

Reply:
moved to the readme @mmacy 

done | addressed from 0d81d8eb456991411256821a7291f7f6a8bb41f0